mods/persistence/modules/cloning/backup_file.dm

```diff
@@ -0,0 +1,31 @@
+#define BASE_CLONE_BACKUP_SIZE 10
+
+/datum/computer_file/data/cloning
+	filetype = "CLN"
+	read_only = TRUE
+	var/mob_age			// Mob's age in years.
```

Comment:
Might be better to save the mobs birthdate? Or maybe save that for a seperate PR that makes players able to select a birthday and such.

Reply:
Separate PR.